controllers/cloud.redhat.com/providers/web/resources_keycloak.go

```diff
@@ -213,9 +370,19 @@ func makeKeycloak(o obj.ClowdObject, objMap providers.ObjectMap, _ bool, nodePor
 	image := provutils.GetKeycloakImage(env)
 
 	c := core.Container{
-		Name:           nn.Name,
-		Image:          image,
-		Env:            envVars,
+		Name:  nn.Name,
+		Image: image,
+		Env:   envVars,
+		Args: []string{
+			"start",
+			"--import-realm",
+			"--hostname-strict",
+			"false",
+			"--http-enabled",
+			"true",
+			"--http-relative-path",
+			"/auth",
```

Comment:
nitpick: it may be more readable to limit the amount of args added to the container. If you agree, can we switch the params to environment variables instead ?
 
According to the [Keycloak docs](https://www.keycloak.org/server/all-config), these should be:

|CLI param| Environment varialbe|
| -------------- | ------------------------------ |
| --hostname-strict | `KC_HOSTNAME_STRICT` |
| --http-enabled | `KC_HTTP_ENABLED` |
| --http-relative-path | `KC_HTTP_RELATIVE_PATH` |

We would still need `start --import-realm` but that'd be it. Again, take this as an optional change.

Reply:
I'd like to separate out the ones that need to be parametrized and the ones that are static. I agree it's a little ugly and if it got bigger maybe we can do it the other way.

Reply:
Sure. it is an implementation detail and I have no strong opinions about it. We can leave it as is.